PyDSS/dssTransformer.py

```diff
@@ -53,23 +53,29 @@ class dssTransformer(dssElement):
     def __init__(self, dssInstance):
         super(dssTransformer, self).__init__(dssInstance)
         self._NumWindings = dssInstance.Transformers.NumWindings()
+        self._dssInstance = dssInstance
 
     @property
     def NumWindings(self):
+        # print(self.dss.Transformers.Name())
+        # print(self.dss.Properties.Value("taps"))
+        # print(self.GetValue('taps'))
         return self._NumWindings
 
     @staticmethod
     def chunk_list(values, nLists):
         return [values[i * nLists:(i + 1) * nLists] for i in range((len(values) + nLists - 1) // nLists)]
 
     def GetValue(self, VarName, convert=False):
+        self.SetActiveObject()
```

Comment:
Similar calls are made in `self.GetVariable` and `self.GetParameter`, so this should not be necessary.  If you prefer the safety and if there is no real cost to doing this, I'm fine with it.

Reply:
Green feeder had a wierd issues which got resolved when I made the changes. I will look into this a bit more to try and figure out the root cause

Reply:
removed